src/Kestrel.Core/Internal/Http/IHttpOutputProducer.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http
     public interface IHttpOutputProducer : IDisposable
     {
         void Abort(ConnectionAbortedException abortReason);
-        Task WriteAsync<T>(Func<PipeWriter, T, long> callback, T state);
+        Task WriteAsync<T>(Func<PipeWriter, T, long> callback, T state, CancellationToken cancellationToken);
```

Comment:
The caller never has a cancellationToken.

Reply:
Can you elaborate? WriteChunkedAsync calls this method today.

Reply:
See above response. I was thinking about the old code.